### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by jaimevillegas**
Hi there!

I have made the following changes to the repository:

- Add JSX elements without extra functionality, to match the design. ⚡️
- Add CSS properties to elements, to add styling matching Zeplin's design. 🎨

Thank you for taking the time to review my code! 🔥🚀

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,4 +1,235 @@
...
+body {
+  width: 100%;
+  background-color: var(--pale-grey);
+}
+
+.App {
+  width: 80%;
+  margin: 100px auto;
+
+  /* padding: 0 0 49.625rem; */
```
Comment (huseyinbiyik) [inline src/App.css]: Please don't leave any uncommented code in your codebase. In this way, you can keep your codebase clean, organized and professional 👍

File: src/App.css [hunk 2]
```diff
@@ -1,4 +1,235 @@
...
+  background-color: var(--white-two);
+  border-bottom: 2px solid var(--pale-grey);
+}
+
+.navbar > div {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-header h1 {
```
Comment (huseyinbiyik) [inline src/App.css:48]: For this project, your code needs to be identical to the Zeplin guideline. Kindly fix the alignment of the logo.
Screenshot 2023-04-06 at 9 23 05 PM
Screenshot 2023-04-06 at 9 23 19 PM

File: src/App.css [hunk 3]
```diff
@@ -1,4 +1,235 @@
...
+  letter-spacing: normal;
+  color: var(--azure);
+}
+
+.nav-header div {
+  display: flex;
+  align-items: center;
+}
+
+.nav-header .nav-link a {
```
Comment (huseyinbiyik) [inline src/App.css:66]: Please use the color code on the design. Normal state is #757575 with 50% opacity. Active link is the same color with 100% opacity.
Screenshot 2023-04-06 at 9 25 31 PM
Screenshot 2023-04-06 at 9 26 59 PM

File: src/App.css [hunk 4]
```diff
@@ -1,4 +1,235 @@
...
+  padding: 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: var(--white-two);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.book-category {
```
Comment (huseyinbiyik) [inline src/App.css:108]: Kindly fix the color value, font-size of the category text. Please notice the opacity 👍 
Screenshot 2023-04-06 at 9 27 21 PM
Screenshot 2023-04-06 at 9 28 03 PM

File: src/App.css [hunk 5]
```diff
@@ -1,4 +1,235 @@
...
+  color: #494949;
+  font-weight: bold;
+}
+
+.book-title {
```
Comment (huseyinbiyik) [inline src/App.css:115]: Please fix the font-color, font-family and font-size of the book title 👍  You can get the font from this l[ink](https://fonts.google.com/specimen/Roboto+Slab).
Screenshot 2023-04-06 at 9 28 53 PM
Screenshot 2023-04-06 at 9 29 22 PM

File: src/App.css [hunk 6]
```diff
@@ -1,4 +1,235 @@
...
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+
+.book-author {
```
Comment (huseyinbiyik) [inline src/App.css:122]: Please fix the font-family, font-size, font-weight, font-color of the author text. Also please fix Comments, Remove and Edit. They have the same value with author.
Screenshot 2023-04-06 at 9 30 30 PM
Screenshot 2023-04-06 at 9 30 48 PM

File: src/App.css [hunk 7]
```diff
@@ -1,4 +1,235 @@
...
+  margin-bottom: 30px;
+}
+
+.add-book-container {
+  width: 90%;
+  margin: 0 auto;
+  padding-bottom: 40px;
+}
+
+.add-book-container h2 {
```
Comment (huseyinbiyik) [inline src/App.css:231]: Please fix the font color and font size of the ADD BOOK heading 👍 
Screenshot 2023-04-06 at 9 33 16 PM
Screenshot 2023-04-06 at 9 33 51 PM

File: src/App.css [hunk 8]
```diff
@@ -1,4 +1,235 @@
...
+  display: flex;
+  justify-content: space-between;
+}
+
+.add-book-container input {
+  font-size: 1.5rem;
+  padding: 5px;
+  border: 1px solid rgb(165, 164, 164);
+  border-radius: 6px;
+}
```
Comment (huseyinbiyik) [inline src/App.css]: Please fix the font-size, border color, font color of input area 👍 
Screenshot 2023-04-06 at 9 35 29 PM
Screenshot 2023-04-06 at 9 36 09 PM
Screenshot 2023-04-06 at 9 38 28 PM
Screenshot 2023-04-06 at 9 39 03 PM

File: src/components/AddBook.jsx [hunk 9]
```diff
@@ -1,25 +1,40 @@
...
-      <input type="text" placeholder="Title" value={title} onChange={(e) => setTitle(e.target.value)} required />
-      <input type="text" placeholder="Author" value={author} onChange={(e) => setAuthor(e.target.value)} required />
-      <button type="submit" onClick={() => dispatch(addBook({ title, author, id }))}>Add book</button>
-    </form>
+    <div className="add-book-container">
+      <h2>ADD NEW BOOK</h2>
+      <form action="">
+        <input type="text" placeholder="Title" value={title} onChange={(e) => setTitle(e.target.value)} required />
+        <input type="text" placeholder="Author" value={author} onChange={(e) => setAuthor(e.target.value)} required />
+        <button type="submit" onClick={(e) => submitBook(e)}>Add book</button>
```
Comment (huseyinbiyik) [inline src/components/AddBook.jsx:36]: Kindly fix the alignment of button. It needs to be aligned as marked with the red line 👍  
Screenshot 2023-04-06 at 9 37 11 PM
Screenshot 2023-04-06 at 9 37 36 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @jaimevillegas 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Design layout implemented very well 👍 
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.jsx (hunk 9 above)
Comment (jaimevillegas) [reply src/components/AddBook.jsx:36]: Hi @huseyinbiyik!

Thank you for all your comments! It helped me to improve my code and styling abilities. I reviewed all your comments, and now my code is according to that!

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @jaimevillegas ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitFlow:+1: 
- Good job styling your project as requested :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**